rubocop-style.yml

```diff
@@ -112,3 +112,23 @@ Style/NumberedParametersLimit:
 # https://docs.rubocop.org/rubocop/cops_style.html#styleselectbyregexp
 Style/SelectByRegexp:
   Enabled: true
+
+# https://docs.rubocop.org/rubocop/cops_style.html#styleopenstructuse
+Style/OpenStructUse:
+  Enabled: true
+
+# https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecrequiremfa
+Gemspec/RequireMFA:
```

Comment:
@jmjurado23 I think this rule should be moved to `rubocop-gemspec.yml`

https://github.com/nosolosoftware/rubocop-nosolosoftware/blob/master/rubocop-gemspec.yml

Reply:
Tienes razón, la he puesto en el fichero que no es